pallets/schemas/src/lib.rs

```diff
@@ -263,6 +282,68 @@ pub mod pallet {
 			Self::deposit_event(Event::SchemaMaxSizeChanged { max_size });
 			Ok(())
 		}
+
+		/// Propose to create a schema.  Creates a proposal for council approval to create a schema
+		///
+		#[pallet::call_index(2)]
+		#[pallet::weight(1000)]
+		pub fn propose_to_create_schema(
+			origin: OriginFor<T>,
+			model: BoundedVec<u8, T::SchemaModelMaxBytesBoundedVecLimit>,
+			model_type: ModelType,
+			payload_location: PayloadLocation,
+		) -> DispatchResult {
+			let proposer = ensure_signed(origin)?;
+
+			let proposal: Box<T::Proposal> = Box::new(
+				(Call::<T>::create_schema_via_governance {
+					creator_key: proposer.clone(),
+					model,
+					model_type,
+					payload_location,
+				})
+				.into(),
+			);
+			T::ProposalProvider::propose_with_simple_majority(proposer, proposal)?;
+			Ok(())
+		}
+
+		/// Create a schema by means of council approval
+		///
+		/// # Events
+		/// * [`Event::SchemaCreated`]
+		///
+		/// # Errors
+		/// * [`Error::LessThanMinSchemaModelBytes`] - The schema's length is less than the minimum schema length
+		/// * [`Error::ExceedsMaxSchemaModelBytes`] - The schema's length is greater than the maximum schema length
+		/// * [`Error::InvalidSchema`] - Schema is malformed in some way
+		/// * [`Error::SchemaCountOverflow`] - The schema count has exceeded its bounds
+		#[pallet::call_index(3)]
+		#[pallet::weight(1000)]
+		pub fn create_schema_via_governance(
+			origin: OriginFor<T>,
+			creator_key: T::AccountId,
+			model: BoundedVec<u8, T::SchemaModelMaxBytesBoundedVecLimit>,
+			model_type: ModelType,
+			payload_location: PayloadLocation,
+		) -> DispatchResult {
+			T::CreateSchemaViaGovernanceOrigin::ensure_origin(origin)?;
```

Comment:
```suggestion
			let eitherGovOrAccount = T::CreateSchemaViaGovernanceOrigin::ensure_origin(origin)?;
			// Assume left is the account
			let creator_key = eitherGovOrAccount.left().unwrap_or(0x0);
```

```
// The origin that is allowed to create providers
	#[cfg(not(feature = "frequency"))]
	type CreateSchemaOrigin = EitherOfDiverse<
		EnsureRoot<AccountId>,
        EnsureSigned<AccountId>
    >;
	#[cfg(feature = "frequency")]
	type CreateSchemaOrigin = EitherOfDiverse<
		EnsureRoot<AccountId>,
		pallet_collective::EnsureProportionMoreThan<AccountId, CouncilCollective, 1, 2>,
	>;
```

Reply:
As per discussion with @wilwade and @shannonwells, we decided to keep the bifurcated design for simplicity and consistency.